web.go

```diff
@@ -83,6 +84,22 @@ func setupRouter() *gin.Engine {
 		}),
 	))
 
+	if !isSecureServer && config.TLSEnforce {
+		r.Use(secure.Secure(secure.Options{
+			AllowedHosts:          []string{},
+			SSLRedirect:           true,
+			SSLHost:               "",
+			SSLProxyHeaders:       map[string]string{"X-Forwarded-Proto": "https"},
+			STSSeconds:            0,
+			STSIncludeSubdomains:  false,
+			FrameDeny:             true,
```

Comment:
I don't know of anyone currently `IFRAME`ing the current device control UI... but this would disable that (intentionally)... might consider a white-list instead?